core/src/components/tab-button/test/shape/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Tab Button - Shape</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
+    <link href="../../../../../css/core.css" rel="stylesheet" />
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet" />
+    <script src="../../../../../scripts/testing/scripts.js"></script>
+    <script nomodule src="../../../../../dist/ionic/ionic.js"></script>
+    <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+  </head>
+
+  <body>
+    <ion-app>
+      <ion-header>
+        <ion-toolbar>
+          <ion-title>Tab Button - Shape</ion-title>
+        </ion-toolbar>
+      </ion-header>
+
+      <ion-content id="content" style="text-align: center">
+        <ion-text>
+          (Only available for ionic theme)
+        </ion-text>
+        <ion-tab-bar style="margin-top: 16px">
+          <ion-text style="flex: 1">
+            Default Shape
+          </ion-text>
+          <ion-text style="flex: 1">
+            Soft Shape
+          </ion-text>
+          <ion-text style="flex: 1">
+            Round Shape
+          </ion-text>
+          <ion-text style="flex: 1">
+            Rectangular Shape
+          </ion-text>
+        </ion-tab-bar>
+
+        <ion-tab-bar>
+          <ion-tab-button id="default-tab-button" style="background: #ff9e99">
```

Comment:
```suggestion
          <ion-tab-button id="default-tab-button">
```

Remove all the styles to the tab button. Use `--background: #f6f6f6;` on tab bar to accomplish the same scenario:

```html
ion-tab-bar {
   --background: #f6f6f6;
}
```

Reply:
My goal was to set a unique background color and icon to distinguish each button and its shape. Setting a background color to the tab-bar will not make the button shapes visible.